static/js/containers/LearnerPage_test.js

```diff
@@ -59,9 +59,14 @@ import {
   clearSelectField,
   GoogleMapsStub,
 } from '../util/test_utils';
+import StaffLearnerInfoCard from '../components/StaffLearnerInfoCard';
+import {
+  REQUEST_DASHBOARD,
+  RECEIVE_DASHBOARD_SUCCESS,
+} from '../actions/dashboard';
 
 describe("LearnerPage", function() {
-  this.timeout(10000);
+  this.timeout(15000);
```

Comment:
Yes, but I had an idea for a change that might make this unnecessary which I forgot to implement, I'll try it now.